src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -400,8 +405,6 @@ def forward(
                         f" {past_key.shape}"
                     )
 
-                past_key_value = (past_key, past_value)
```

Comment:
`past_key_value` is now a `Cache` instance that is update in place with the `.update()` function (L413 in the updated file)